src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -968,3 +971,60 @@
 type: command
 short-summary: Update metadata for an IoT Central application.
 """
+
+helps['iot hub private-endpoint-connection approve'] = """
```

Comment:
highly recommend shorter names for these

Reply:
particularly the `private-endpoint-connection` ,  there is too much to type

Reply:
I tend to agree, however the documentation from the CLI about supporting these commands specifically states those names, so I'm not sure how much leeway we have:

```
- The parent resource should expose a single command group called `private-endpoint-connection` with four commands: `approve`, `reject`, `delete`, `show`.
```

```
- The parent resource should expose a single command group called `private-link-resource` with one commands: `list`.
```

Reply:
[Here's a link to the documentation](https://github.com/Azure/azure-cli/blob/dev/doc/private_endpoint_connection_command_guideline.md)

Reply:
Ok thanks - didn't realize there was an Azure CLI wide spec for this already.